src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/casenotes/CaseNotesSynchronisationService.kt

```diff
@@ -59,17 +60,12 @@ class CaseNotesSynchronisationService(
     }
     caseNotesMappingService.getMappingGivenNomisIdOrNull(event.caseNoteId)
       ?.also { mapping ->
-        caseNotesService.upsertCaseNote(nomisCaseNote.toDPSUpdateCaseNote(mapping.dpsCaseNoteId))
-//     "caseNoteId").isEqualTo(casenote1.id)
-//     "bookingId").isEqualTo(bookingId)
-//     "caseNoteType.code").isEqualTo("ALL")
-//     "caseNoteSubType.code").isEqualTo("SA")
-//     "authorUsername").isEqualTo("JANE.NARK")
-//     "prisonId").isEqualTo("BXI")
-//     "caseNoteText").isEqualTo("A note")
-//     "amended").isEqualTo("false")
-//     "occurrenceDateTime") assertThat(LocalDateTime.parse(it)).isCloseTo(now, within(2, ChronoUnit.MINUTES)) }
-
+        caseNotesService.upsertCaseNote(
+          nomisCaseNote.toDPSSyncCaseNote(
+            event.offenderIdDisplay!!,
```

Comment:
probably worth making event.offenderIdDisplay mandatory - I guess it can't be null without a coding error?

Reply:
ok will do next time